src/server/signaling_server.js

```diff
@@ -139,7 +139,10 @@ io.on('connection', async (socket) => {
   });
 
   socket.on('new-message', ({ id, message }) => {
-    const newMessage = `${users[id]}: ${message}`;
```

Comment:
Please change this back to the string. You must not have merged the latest main into your branch.